tests/rules/javascript/test.event_listener_fourth.js

```diff
@@ -0,0 +1,128 @@
+import JavaScriptScanner from 'scanners/javascript';
+import { VALIDATION_WARNING } from 'const';
+import { singleLineString } from 'utils';
+import * as messages from 'messages';
+
+describe('event_listener_fourth', () => {
+  it('should not allow a true literal', () => {
+    var code = 'window.addEventListener("click", function(){}, false, true);';
```

Comment:
I agree, in fact, I'm beginning to wonder if there may be a better way to wrap ESLint here. For instance when we are looking for call expressions for something like `foo()` we normally are looking for `window.foo()` or `var a = foo; a();` or other sort of things. And I'm seeing that the rules are often having to contain this logic individually and some of it is starting to get repeated.